extensions/mssql/src/contracts.ts

```diff
@@ -1021,6 +1022,7 @@ export interface SqlAssessmentResult extends azdata.ResultStatus {
 
 export interface SqlMigrationAssessmentParams {
 	ownerUri: string;
+	databases?: string[];
```

Comment:
This is required on the STS side to actually do anything, any reason you have it optional here?